internal/http/services/owncloud/ocdav/move.go

```diff
@@ -218,6 +218,13 @@ func (s *svc) handleMove(ctx context.Context, w http.ResponseWriter, r *http.Req
 			return
 		}
 
+		// do not allow overwriting spaces
+		if dstStatRes.GetInfo().GetPath() == "." {
```

Comment:
Is that enough? Don't we need to check for
```suggestion
		if dstStatRes.GetId().GetSpaceId() == dstStatRes.GetId().GetOpaqueId() && dstStatRes.GetInfo().GetPath() == "." {
```